File: Veterinario/VeterinarioBlueJ/Mascota.java
public class Mascota {
    private String numeroCarnet;
    private String nombre;
    private String especie;
    private String raza;
    private int edad;
    private Encargado encargado; // Asociación/Composición

    public Mascota(String numeroCarnet, String nombre, String especie, String raza, int edad, Encargado encargado) {
        this.numeroCarnet = numeroCarnet;
        this.nombre = nombre;
        this.especie = especie;
        this.raza = raza;
        this.edad = edad;
        this.encargado = encargado;
    }

    public String getNumeroCarnet() { return numeroCarnet; }
    public String getNombre() { return nombre; }
    public String getEspecie() { return especie; }
    public String getRaza() { return raza; }
    public int getEdad() { return edad; }
    public Encargado getEncargado() { return encargado; }
}

File: Veterinario/ConsultorioVeterinario-V2/ConsultorioVeterinario/Logica/Mascota.cs
namespace ConsultorioVeterinario.Logica
{
    public class Mascota
    {
        public string NumeroCarnet { get; set; }
        public string Nombre { get; set; }
        public string Especie { get; set; }
        public string Raza { get; set; }

        public int Edad { get; set; }
        public Encargado Encargado { get; set; } 

        public Mascota(string carnet, string nombre, string especie, Encargado encargado)
        {
            NumeroCarnet = carnet;
            Nombre = nombre;
            Especie = especie;
            Encargado = encargado;
        }
    }
}